Playbooks/playbook-Generate_log-bundle.yml

```diff
@@ -0,0 +1,139 @@
+id: Generate log-bundle
+version: -1
+name: Generate log-bundle
```

Comment:
Change name to: Generate Log Bundle - Demisto (verify with @kirbles19 )

Reply:
I would call it `Send Logs to Demisto`